docs/monitoring/README.md

```diff
@@ -0,0 +1,49 @@
+# Monitoring
+
+Steward provides some metrics which can be collected with [Prometheus].
+There is also a [example dashbord] for [Grafana] available to display the metrics.
+
+## Tenant Controller Metrics
+
+| name | type | description |
+| ---- | ---- | ----------- |
+| steward_tenant_total_number | gauge | number of tenants in the cluster | 
+ 
+## Pipelinerun Controller Metrics
+
+| name | type | label | description |
+| ---- | ---- | ----- | ----------- |
+| steward_pipeline_runs_started_total_count   | counter   | _none_ | counter is increased by every started pipeline run |
+| steward_pipeline_runs_completed_total_count | counter   | result | counters with result label are increased when result of pipeline run is set |
+| steward_pipeline_runs_completed_total_count | histogram | state  | histogram with 15 exponential buckets starting from 125ms with factor 2 for the different pipelinerun states |
+
+## Example Installation with Prometheus Operator
+
+### Prerequisites 
+-   Environment variable `$KUBECONFIG` set
+-   Helm 2 (>= 2.14) installed locally
+-   Helm initialized ([Tiller is installed][tiller-install] on the target cluster)
+
```

Comment:
Add this requirement:

- Current directory is `docs/monitoring/` (where this README is located)